test-suite/handwritten-src/java/android/os/Parcel.java

```diff
@@ -0,0 +1,155 @@
+package android.os;
```

Comment:
Did you write this from scratch, or copy it from somewhere?  It needs a license, and/or a comment explaining it's purpose, particularly pointing out that this is a mock replacement for Android's implementation.